src/us/kbase/test/auth2/lib/AuthenticationIdentityProviderTest.java

```diff
@@ -100,131 +98,4 @@ public void failGetProviders() throws Exception {
 					"accessible in the storage system"));
 		}
 	}
-	
```

Comment:
These tests are now part of the authentication link & login tests, since the `getIdentityProviderURL` method is no more